examples/inline_bot/src/main.rs

```diff
@@ -0,0 +1,73 @@
+use teloxide::{Bot, payloads::AnswerInlineQuery, prelude::*, types::{InlineQueryResult, InlineQueryResultArticle, InputMessageContent, InputMessageContentText}};
+use tokio_stream::wrappers::UnboundedReceiverStream;
+
+#[tokio::main]
+async fn main() {
+    run().await;
+}
+
+async fn run() {
+    let bot = Bot::from_env();
+    // Create a new dispatcher to handle incoming messages
+    let dp = Dispatcher::new(bot);
+    dp.inline_queries_handler(|rx: DispatcherHandlerRx<Bot, InlineQuery>| {
```

Comment:
We usually don't assign `Dispatcher` to a variable before adding handlers, see e.g. https://github.com/teloxide/teloxide/blob/dev/examples/sqlite_remember_bot/src/main.rs#L45-L46.

Reply:
```suggestion
    let dp = Dispatcher::new(bot);
    dp.inline_queries_handler(|rx: DispatcherHandlerRx<AutoSend<Bot>, InlineQuery>| {
```